assets/src/stories-editor/components/resizable-box/index.js

```diff
@@ -215,40 +249,84 @@ class EnhancedResizableBox extends Component {
 						lastDeltaW = deltaW;
 					}
 
-					if ( ! angle ) {
-						// If the resizing is to left or top then we have to compensate
-						if ( REVERSE_WIDTH_CALCULATIONS.includes( direction ) ) {
-							const leftInPx = getPixelsFromPercentage( 'x', parseFloat( blockElementLeft ) );
-							blockElement.style.left = getPercentageFromPixels( 'x', leftInPx - lastDeltaW ) + '%';
+					const dimensions = blockElement.getBoundingClientRect();
```

Comment:
Hmm, what if the block is rotated?
`blockElement = element.closest( '.wp-block' ).parentNode;` meaning that it's not considering the rotation since the parent node isn't rotated, only `.wp-block` is. If that's the case then this would mean that `actualRight` and `actualLeft` would probably be incorrect. Thoughts?

Reply:
Fixed in 9e42965816ec3a77e4b1469983b2290f2104ccef.